main_logic/tts_client.py

```
@@ -990,3 +1007,241 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
     else:
         logger.error(f"{core_api_type}不支持原生TTS，请使用自定义语音")
         return dummy_tts_worker
+
+
+def local_cosyvoice_worker(request_queue, response_queue, audio_api_key, voice_id):
```

Ruff (0.14.10)

Unused function argument: `audio_api_key`

(ARG001)